### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MishaLehotskyi**
- [DEMO LINK](https://MishaLehotskyi.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://MishaLehotskyi.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+  font-style: normal;
```
Comment (volodymyr-soltys97) [inline src/style.css:12]: This is default value, no need to write it
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
No need to change styles when focusing
image

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+* {
+  box-sizing: border-box;
+}
```
Comment (Esceype) [inline src/style.css:15]: Move it to body selector

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+.html {
+  font-family: "Avenir", Arial, sans-serif;
```
Comment (Esceype) [inline src/style.css:18]: Add one more font as a fallback font

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  left: 13px;
+}
+
+.search-bar__input {
+  width: 100%;
+  font-family: "Avenir", Arial, sans-serif;
+  font-weight: 700;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
```
Comment (Esceype) [inline src/style.css:56]: Move this two colors to CSS variables

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-bar__input:focus {
+  outline: none;
+  color: #3d4e61;
```
Comment (Esceype) [inline src/style.css:66]: Also move it to CSS variable and reuse it

File: src/index.html [hunk 6]
```diff
@@ -11,6 +11,35 @@
...
+        class="search-bar__form search-bar__form--small"
+        action="#"
+        method="get"
+        data-qa="small"
+      >
+        <input
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="search-bar__input search-bar__input--small"
+          type="text"
+          name="search"
```
Comment (Esceype) [inline src/index.html:40]: Provide better naming for each name attribute in your inputs

**Review by Esceype — CHANGES_REQUESTED** [review]
GJ! Let`s improve some moments in your code

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-bar__input {
+  width: 100%;
+  box-sizing: border-box;
+  font-family: "Avenir", Arial, sans-serif;
```
Comment (Andriy-Shymkiv) [inline src/style.css:55]: I don't think that you need to specify `font-family` here. you already did it for all html above

**Review by Andriy-Shymkiv — APPROVED** [review]
good job! left one comment. approved in advance

**Pull request closed without merging**